consumatio/external/db/db.py

```diff
@@ -135,7 +135,7 @@ def media_data_exists(self: object, user_id: int, media: str,
             logger.info("media_data entry exists in database")
             return True
 
-    def media_Data(self: object, user_id: int, media: str, media_id: int,
+    def media_data(self: object, user_id: int, media: str, media_id: int,
```

Comment:
Please also change the code where the function is called. Like this, the application crashes